addons/l10n_de/models/chart_template.py

```diff
@@ -16,4 +16,37 @@ def _load(self, company):
                 'external_report_layout_id': self.env.ref('l10n_din5008.external_layout_din5008').id,
                 'paperformat_id': self.env.ref('l10n_din5008.paperformat_euro_din').id
             })
+
+            outstanding_receipt = company.account_journal_payment_debit_account_id
+            outstanding_payment = company.account_journal_payment_credit_account_id
+
+            outstanding_receipt['tag_ids'] = self.env.ref('l10n_de.tag_de_asset_bs_B_II_4')
+            outstanding_payment['tag_ids'] = self.env.ref('l10n_de.tag_de_asset_bs_B_II_4')
+
+        return res
+
+    def _prepare_transfer_account_template(self):
+        res = super(AccountChartTemplate, self)._prepare_transfer_account_template(None)
+        if self in [
+            self.env.ref('l10n_de_skr03.l10n_de_chart_template', raise_if_not_found=False),
+            self.env.ref('l10n_de_skr04.l10n_chart_de_skr04', raise_if_not_found=False)
+        ]:
+            tag_ids = res.get('tag_ids', [])
+            tag_ids.append((4, self.env.ref('l10n_de.tag_de_asset_bs_B_II_4').id))
+            res['tag_ids'] = tag_ids
+
         return res
+
+    def _create_liquidity_journal_suspense_account(self, company, code_digits):
+        if self not in [
+            self.env.ref('l10n_de_skr03.l10n_de_chart_template', raise_if_not_found=False),
+            self.env.ref('l10n_de_skr04.l10n_chart_de_skr04', raise_if_not_found=False)
+        ]:
+            return super()._create_liquidity_journal_suspense_account(company, code_digits)
+        return self.env['account.account'].create({
+            'name': _("Bank Suspense Account"),
```

Comment:
Are we supposed to translate that ?

Reply:
Don't really know if it's a necessity but this is how they have done it in account/models/chart_template.py. And i found it strange to have a COA full in a language and juste some lines in english ?
![image](https://user-images.githubusercontent.com/109144554/228771012-51dbf6a2-f32e-4ba6-abe6-c07b01045885.png)